src/components/article/Tile.tsx

```diff
@@ -23,6 +23,7 @@ const Tile = styled.div`
 
   & > a {
     margin: 0 auto 0 0;
+    color: ${theme.primary};
```

Comment:
Apply only there.primary when user clicked link -> underline color, In other case it must be white. We need to follow 60% 30% 10% convention in our app with colors